awx/main/migrations/0186_djy.py

```diff
@@ -0,0 +1,25 @@
+from django.db import migrations, models
+
+import logging
+
+logger = logging.getLogger('awx.main.migrations')
+
+
+def set_peers_from_control_nodes_true(apps, schema_editor):
+    Instance = apps.get_model('main', 'Instance')
+
+    for instance in Instance.objects.all():
```

Comment:
this block can be written a little more succinctly by using objects.filter(..) followed by a .update(..)